youtube_dl/extractor/nova.py

```diff
@@ -104,35 +132,35 @@ def _real_extract(self, url):
         config = self._download_json(
             config_url, display_id,
             'Downloading config JSON',
-            transform_source=lambda s: re.sub(r'var\s+[\da-zA-Z_]+\s*=\s*({.+?});', r'\1', s))
+            transform_source=lambda s: s[s.index('{'):s.rindex('}') + 1])
 
         mediafile = config['mediafile']
         video_url = mediafile['src']
-
-        m = re.search(r'^(?P<url>rtmpe?://[^/]+/(?P<app>[^/]+?))/&*(?P<playpath>.+)$', video_url)
-        if m:
-            formats = [{
-                'url': m.group('url'),
-                'app': m.group('app'),
-                'play_path': m.group('playpath'),
-                'player_path': 'http://tvnoviny.nova.cz/static/shared/app/videojs/video-js.swf',
-                'ext': 'flv',
-            }]
-        else:
-            formats = [{
-                'url': video_url,
-            }]
-        self._sort_formats(formats)
+        ext = determine_ext(video_url)
+        video_url = video_url.replace('&{}:'.format(ext), '')
 
         title = mediafile.get('meta', {}).get('title') or self._og_search_title(webpage)
-        description = self._og_search_description(webpage)
+        description = clean_html(self._og_search_description(webpage, default=None))
```

Comment:
The third test: http://novaplus.nova.cz/porad/policie-modrava/video/5591-policie-modrava-15-dil-blondynka-na-hrbitove/.